Fenced_Frames_Ads_Reporting.md

```diff
@@ -262,6 +264,25 @@ window.fence.setReportEventDataForAutomaticBeacons({
 });
 ```
 
+#### Cross-Origin Support
+
+Data for automatic beacons can only be set by documents that are same-origin to the mapped URL of the fenced frame config. However, cross-origin documents can still send automatic beacons, if the document and the data are **both** opted in.
+
+A cross-origin document will be considered opted into sending automatic beacons if it is served with the response header `Allow-Fenced-Frame-Automatic-Beacons: true`.
+
+To opt in the data, the dictionary passed into `setReportEventDataForAutomaticBeacons` takes an optional `crossOriginExposed` boolean that defaults to false. If set to true, the automatic beacon data can be used if a cross-origin document wants to send an automatic beacon and is opted in. A document will use the data of the first ancestor frame that has automatic beacon data registered for the event type being sent.
+
+```
+window.fence.setReportEventDataForAutomaticBeacons({
+  'eventType': 'reserved.top_navigation_start',
+  'eventData': 'an example string',
+  'destination': ['seller', 'buyer'],
+  'crossOriginExposed': true,
+});
+```
+
```

Comment:
If this change is also applied to component ads (i.e. nested fenced frames), we should update this section as well: https://github.com/WICG/turtledove/blob/main/Fenced_Frames_Ads_Reporting.md#:~:text=The%20ad%20component%20must%20still%20opt%20in%20using%20setReportEventDataForAutomaticBeacons%20before%20the%20beacon%20can%20send.